### PR TITLE
Con 393 week 3 homepage changes

### PR DESCRIPTION
**Pull request by Ashley-Bass**

### CODE REVIEW

File: src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/Index.cshtml [hunk 1]
```diff
@@ -27,3 +49,32 @@
...
+                <ul class="govuk-list das-list--featured">
+                    <li><a href="@("/commitments/accounts/"+Model.Data.HashedAccountId+"/apprentices/home")" class="govuk-link govuk-link--no-visited-state">Add an apprentice</a></li>
+                    <li><a href="@("/commitments/accounts/"+Model.Data.HashedAccountId+"/apprentices/home")" class="govuk-link govuk-link--no-visited-state">Manage your apprentices</a></li>
+                    <li><a href="@("/accounts/"+Model.Data.HashedAccountId+"/providers")" class="govuk-link govuk-link--no-visited-state">Add a new training provider</a></li>
+                    <li><a href="@("/accounts/"+Model.Data.HashedAccountId+"/agreements")" class="govuk-link govuk-link--no-visited-state">Add a new organisation</a></li>
+                    <li><a href="@("/accounts/"+Model.Data.HashedAccountId+"/rename")" class="govuk-link govuk-link--no-visited-state">Rename your account</a></li>
+                </ul>
+            </div>
+            <div class="das-dashboard-footer__column">
+                <a class="das-promo" href="#">
```
Comment (Stephek) [inline src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/Index.cshtml]: Add a link here if one exists in the existing home page.

File: src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/Index.cshtml [hunk 2]
```diff
@@ -27,3 +49,32 @@
...
+                    What you need to know about apprenticeships
+                </a>
+                <p>
+                    Find out more about <a href="#" class="govuk-link govuk-link--no-visited-state"> what the Apprenticeship Service can do for your organisation</a>.
```
Comment (Stephek) [inline src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/Index.cshtml]: Add a link here if one exists in the existing home page.

File: src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/Index.cshtml (hunk 1 above)
Comment (Ashley-Bass) [reply src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/Index.cshtml]: No current similar link exists

File: src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/Index.cshtml (hunk 2 above)
Comment (Ashley-Bass) [reply src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/Index.cshtml]: No current similar link exists

File: src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/Index.cshtml [hunk 3]
```diff
@@ -12,7 +12,29 @@
...
+            </div>
+        </div>
+    </div>
+</section>
+
+
+<main class="govuk-main-wrapper das-section--dashboard" id="main-content" role="main">
     <div class="govuk-width-container">
         <div class="das-panels">
             <div class="das-panels__row">
```
Comment (Stephek) [inline src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/Index.cshtml]: Should be  <div class="das-panels__col das-panels__col--primary">

Comment (Ashley-Bass) [reply src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/Index.cshtml]: 👍

File: src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/Index.cshtml [hunk 4]
```diff
@@ -5,25 +5,76 @@
...
+    <div class="govuk-width-container">
+        <div class="govuk-grid-row">
+            <hr class="das-section-break das-section-break--l das-section-break--heavy das-section-break--gutters">
+            <div class="das-dashboard-footer__column das-dashboard-footer__column--title">
+                <h2 id="more-title" class="das-dashboard-footer__title">More on the Apprenticeships Service</h2>
+            </div>
+            <div class="das-dashboard-footer__column">
+                <h3 class="govuk-heading-m das-dashboard-footer__sub-title">Most active</h3>
+                <ul class="govuk-list das-list--featured">
+                    <li><a href="@("/commitments/accounts/"+Model.Data.HashedAccountId+"/apprentices/home")" class="govuk-link govuk-link--no-visited-state">Add an apprentice</a></li>
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/Index.cshtml]: top two links need to have the commitments base url prepended and the 3rd one needs Provider relationships base url

Comment (Ashley-Bass) [reply src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/Index.cshtml]: Done

**Pull request merged**